README.md

```
@@ -25,13 +25,41 @@
 - [x] 인기 테마를 조회한다.
-    - [x] 최근 일주일을 기준으로 하여 해당 기간 내에 방문하는 예약이 많은 테마 10개를 확인하려 합니다.
-    - [x] 예를 들어 오늘이 4월 8일인 경우, 게임 날짜가 4월 1일부터 4월 7일까지인 예약 건수가 많은 순서대로 10개의 테마를 조회할 수 있어야 합니다.
-    - [x] / 요청 시 인기 테마 페이지를 응답합니다
+    - [x] 최근 일주일을 기준으로 하여 해당 기간 내에 방문하는 예약이 많은 테마 10개를 확인할 수 있다.
+    - [x] 예를 들어 오늘이 4월 8일인 경우, 게임 날짜가 4월 1일부터 4월 7일까지인 예약 건수가 많은 순서대로 10개의 테마를 조회할 수 있다.
+    - [x] / 요청 시 인기 테마 페이지를 응답한다
+
+## 3.5단계 - 회원가입 기능 - 나중
+
+- [ ] 회원가입을 할 수 있다.
+    - [ ] 사용자가 이름, 이메일, 비밀번호를 입력하여 회원가입을 할 수 있다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)